lux/extensions/admin/sidebar.py

```diff
@@ -26,6 +26,9 @@ def add_css(all):
     sidebar.toggle.border = '1px solid #ddd'
     sidebar.info.p.color = '#ccc'
     sidebar.info.background = '#425466'
+    sidebar.menu.max_height = px(500)
```

Comment:
We need to explicitly set `max-height` to apply smooth transition, should I change 500px to 999px?